contracts/proxy/ERC1967/ERC1967Upgrade.sol

```diff
@@ -123,7 +123,7 @@ abstract contract ERC1967Upgrade is IERC1967 {
      * https://eth.wiki/json-rpc/API#eth_getstorageat[`eth_getStorageAt`] RPC call.
      * `0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103`
      */
-    function _getAdmin() internal view returns (address) {
+    function _getAdmin() internal view virtual returns (address) {
```

Comment:
When `ERC1967Upgrade` becomes a library, this will be modifier will have to dissapear.

Reply:
Yes, in the meantime are we ok?
I'd like to help getting #4204 merged if it becomes a blocker.

Reply:
It really depends which PR gets merged first. I'm ok with your approach, I'm just saying that whichever PR gets merge first, it will create conflicts on the other.

The alternative would be to make this non virtual, and use a different name for the internal getter for the immutable variable (or not have a getter and make the immutable variable internal)

Reply:
Solving since #4204 went first. Thanks!